src/opendr/perception/multimodal_human_centric/rgbd_hand_gesture_learner/README.md

```diff
@@ -0,0 +1,3 @@
+# OpenDR - RGBD Hand Gesture Recognition method 
+
+This module provides the baseline implementation of the hand gesture recognition method using color and depth image via early fusion: the aligned RGB and Depth images are concatenated and fed into a convolutional neural network to predict the type of static hand gestures. Different convolutional neural networks architecture with ImageNet pretrained weights are supported. In addition, we also provide a light-weight pretrained hand gesture recognition model based on the `mobilenet_v2` architecture, which recognizes a set of 16 different static hand gestures defined in [MEGURU: a gesture-based robot program builder for Meta-Collaborative workstations](https://www.sciencedirect.com/science/article/abs/pii/S0736584520302957).
```

Comment:
Is it possible to also add a link to a code repository or webpage from where the files in `algorithm/architectures` are copied?
If I understand correctly the BSD 3-clause, you should also mention the original author of the code in the documentation page of the learner.

Reply:
I have added the reference to the original code